packs/orion/actions/ncm_config_download.py

```diff
@@ -27,6 +27,10 @@ def run(self, node, platform, configs):
         results = {}
         node_ids = []
 
+        # Split into an Array, as enum and array type does not play nice
+        # in the WebUI.
+        configs = configs.split(",")
```

Comment:
I've checked and customer config types are possible now in NCM, so the enum should not be present. So I've changed back to an array and dropped the enum.